test/make.sh

```diff
@@ -80,20 +107,27 @@ function check_trailing_whitespace() {
   echo "The following lines have trailing whitespace"
   grep -r '[[:blank:]]$' --exclude-dir=".terraform" --exclude-dir=".kitchen" --exclude="*.png" --exclude="*.pyc" --exclude-dir=".git" .
   rc=$?
-  if [ $rc = 0 ]; then
-    exit 1
+  if [[ ${rc} -eq 0 ]]; then
+    return 1
   fi
 }
 
 function generate_docs() {
   echo "Generating markdown docs with terraform-docs"
-  TMPFILE=$(mktemp)
-  #shellcheck disable=2006,2086
-  for j in `for i in $(find . -type f | grep \.tf$) ; do dirname $i ; done | sort -u` ; do
-    terraform-docs markdown "$j" > "$TMPFILE"
-    python helpers/combine_docfiles.py "$j"/README.md "$TMPFILE"
-  done
-  rm -f "$TMPFILE"
+  local path tmpfile
+  while read -r path; do
+    # shellcheck disable=SC2119
+    tmpfile="$(maketemp)"
+    echo "terraform-docs markdown ${path}"
+    terraform-docs markdown "${path}" > "${tmpfile}"
+    if ! [[ -e "${path}/README.md" ]]; then
```

Comment:
I'm not sure this is the right approach - if there isn't already a README, we should just not generate docs instead of creating a README.

Reply:
@aaron-lane @adrienthebo What do you think?

Reply:
I agree with you Morgante, we could just as easily skip over a directory with no readme.  I think that's best, with a clear output message to the console like `"Skipping ${path} because no README.md exists"`

The reason I took this approach of creating the readme is because that's what the current behavior is, so what you're seeing is an attempt to follow the spirit and intent.

Reply:
In that case, I'm happy to merge as is but please provide a follow-up PR which changes that behavior.

Reply:
Done deal, behavior of skipping directories without README.md files is at: https://github.com/terraform-google-modules/terraform-google-startup-scripts/pull/6